tests/getenclave_extend/README.md

```diff
@@ -0,0 +1,5 @@
+This directory tests enclave creation with different type and flags, with a debug enabled image. It has the following tests:
```

Comment:
Nit: Change this to now say "with both debug and non-debug images"

Reply:
Done. Thanks.